openhtf/plugs/__init__.py

```diff
@@ -184,19 +185,29 @@ class PlugManager(object):
   main framework thread anyway.
 
   Attributes:
-    plug_map: Dict mapping plug type to instantiated plug.
+    _plug_map: Dict mapping plug type to instantiated plug.
   """
 
-  def __init__(self, plug_types):
-    self.plug_map = {}
+  def InitializePlugs(self, plug_types):
+    self._plug_map = {}
     try:
       for plug_type in plug_types:
-        self.plug_map[plug_type] = plug_type()
+        self._plug_map[plug_type] = plug_type()
     except Exception:  # pylint: disable=broad-except
       _LOG.exception('Exception insantiating plug type %s', plug_type)
       self.TearDownPlugs()
       raise
 
+  @property
+  def _asdict(self):
```

Comment:
it's included in TestState->PhaseData. The templates refer to measurements inside of PhaseData, but not the plugs, so I could remove them for now. Though I'm pretty sure Joe's frontend would use them